dashboard/streamlit_app.py

```diff
@@ -139,6 +144,44 @@ def get_judge_type_selection(conn: connect, key: str) -> st.selectbox:
 
     return judge_selection
 
+
+def compile_inputs_as_dict(conn: connect, judge_type: str = None, circuits: str = None,
+                           gender: str = None, date: tuple = None, judge: str = None) -> dict:
+    """Returns input widget returns as a single dictionary"""
+
+    with conn.cursor() as cur:
+        if judge_type:
+            query_type = """
+                            SELECT judge_type_id
+                            FROM judge_type
+                            WHERE type_name = %s
+                            """
+            cur.execute(query_type, [judge_type])
+            judge_type = cur.fetchone()["judge_type_id"]
+
+        if circuits:
+            placeholders = ', '.join([f"'{circuit}'" for circuit in circuits])
+            st.write(placeholders)
+            query_circuit = f"""
+                            SELECT circuit_id
+                            FROM circuit
+                            WHERE name IN ({placeholders})
```

Comment:
Is `IN` better than using `ILIKE` here?

Reply:
I think we have to use `IN` because we are passing in a list to check instead of one single string